game-core/src/main/java/tools/map/making/PlacementPicker.java

```diff
@@ -1,5 +1,11 @@
 package tools.map.making;
 
+import static com.google.common.base.Preconditions.checkState;
+import static tools.util.ToolArguments.TRIPLEA_MAP_FOLDER;
+import static tools.util.ToolArguments.TRIPLEA_UNIT_HEIGHT;
+import static tools.util.ToolArguments.TRIPLEA_UNIT_WIDTH;
+import static tools.util.ToolArguments.TRIPLEA_UNIT_ZOOM;
```

Comment:
Do we have any written conventions on static imports?
I'd like for us to avoid them; it can make refactoring more difficult and I at least find it more confusing in code "where did this variable come from?"

Refactoring is more difficult as if we move the constants, then we need to redo N imports instead of just fixing 1.

Reply:
I was just following the convention established by `CliProperties` (and to avoid increasing the size of the diff further :smile:).  Personally, I prefer non-static imports of constants (except maybe in test contexts), so I can change it, if you like.